kahuna/public/stylesheets/main.css

```diff
@@ -306,32 +306,70 @@ textarea.ng-invalid {
 /* ==========================================================================
    Top bar
    ========================================================================== */
+@media screen and (max-width: 700px) {
+    .top-bar-item.top-bar-item--surplus {
```

Comment:
The point of BEM is to avoid specificity and the modifiers should work using the cascade of the stylesheet.
i.e. `.top-bar-item--surplus {`.